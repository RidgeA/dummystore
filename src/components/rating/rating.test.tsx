import React from "react";
import * as TestRenderer from "react-test-renderer";
import Rating from "./rating";

describe('Rating', () => {
    it('renders', () => {
        const tree = TestRenderer
            .create(
                <Rating value={1}></Rating>
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});