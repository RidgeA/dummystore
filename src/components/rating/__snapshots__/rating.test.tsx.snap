// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating renders 1`] = `
<div
  className="stars"
  style={
    {
      "--rating": 1,
    }
  }
/>
`;
